container.go

```diff
@@ -80,6 +81,10 @@ func (c *Container) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		}
 	}
 
+	if c.StopTimeout == 0 {
+		c.StopTimeout = 10
```

Comment:
The default should be set in replicated so it's baked into the release and doesn't fluctuate with new releases on libyaml